Tasks/MavenV3/task.json

```diff
@@ -227,6 +227,9 @@
             "visibleRule": "javaHomeSelection = JDKVersion",
             "options": {
                 "default": "default",
+                "1.14": "JDK 14",
```

Comment:
What about 1.15 and 1.16 ?

Reply:
@sfragata these are currently not available on hosted images - you can find current available tools in https://github.com/actions/virtual-environments/tree/main/images
Please feel free to open an issue on https://github.com/actions/virtual-environments for pre-installed tools updates on hosted agents.

Reply:
done @anatolybolshakov 
https://github.com/actions/virtual-environments/issues/3693

Reply:
@anatolybolshakov , @alexander-smolyakov , @max-zaytsev , @EzzhevNikita , I don't recommend to deploy these changes to customers.
Policy for Hosted image is **installing only LTS versions of Java (8, 11, 17)**.
Currently, some Hosted images contain non-LTS versions like 12, 13 but they were pre-installed before our policy and we can't remove it without breaking changes. But for all future installations - our policy is only LTS.

Current situation is the following:
- Windows: 8, 11, 13
- Ubuntu 1604, 1804: 8, 11, 12
- Ubuntu 2004: 8, 11
- macOS: 8, 11, 12, 13, 14

Adding Java 12, 13, 14 to dropdown will cause customers confusion because task won't work with these options everywhere.

Reply:
@maxim-lobanov thanks for the info, we will revert these changes then.